src/main/java/seedu/address/model/person/BirthdayMonth.java

```diff
@@ -35,7 +37,11 @@ public BirthdayMonth(String birthdayMonth) {
 
         assert integerValue > 0 && integerValue < 13;
 
-        monthString = monthStrings[integerValue - 1];
+        // Generate birthday cake emoji
+        byte[] emojiByteCode = new byte[] {(byte) 0xF0, (byte) 0x9F, (byte) 0x8E, (byte) 0x82};
+        String emoji = new String(emojiByteCode, StandardCharsets.UTF_8);
+
+        monthString = emoji + " " + monthStrings[integerValue - 1];
```

Comment:
This makes the entire UI more creative and visually appealing to the users as compared to before